config/suppressions.xml

```diff
@@ -10,6 +10,8 @@
     <!-- don't validate generated files -->
     <suppress checks="RegexpSingleline" files="releasenotes\.xml"/>
 
+    <suppress checks="RegexpMultiline" files="config_whitespace\.xml" lines="244"/>
```

Comment:
Please think about how to avoid this suppression, it is pain to support suppression based by line number. Can we change example ?

Reply:
done

Reply:
done